### PR TITLE
Fix errors in crawler and links

### PR DESCRIPTION
**Pull request by bidoubiwa**
This PR is mainly used to generate a netlify link to test the docs-scraper! If the issue is fixed on docs-scraper (or here) I will fix the errors not showing in the searchbar or in the navigation

### CODE REVIEW

**Comment by react-learner** [comment]
@bidoubiwa Clicking on Installation still brings you to the Reference site section. I think it'll be fixed if you `git mv reference/features/installation.md learn/tutorials/`. (Also, the other articles in Feature References sidebar are in alphabetical order, so Errors is breaking the pattern 😱😂)

Other than that this looks great! If you rebase I will merge 🙌

**Comment by react-learner** [comment]
In the long run, it could be nice to have FAQ, Errors list, and other common resources that don't quite fit in any other category in a drop-down menu from the top navbar. That way you could navigate to them from anywhere on the site.

**Comment by bidoubiwa** [comment]
> drop-down menu from the top navbar

This should be investigated! I dont know if vuepress offers an easy solution to this.

File: reference/features/README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Understanding our reference documentation is easier with background knowledge of
 [Faceted Search](/reference/features/faceted_search.md)
 [Field Properties](/reference/features/field_properties.md)
 [Filtering](/reference/features/filtering.md)
-[Installation](/reference/features/installation.md)
+[Installation](/learn/tutorials/installation.md)
```
Comment (react-learner) [inline reference/features/README.md:18]:
```suggestion
```

**Review by react-learner — APPROVED** [review]
Awesome! Thanks for remembering the redirect! 🔥

**Pull request merged**